src/analysis/normed_space/operator_norm.lean

```diff
@@ -1257,29 +1394,40 @@ end
 
 /-- The non-negative norm of the tensor product of a scalar linear map and of an element of a normed
 space is the product of the non-negative norms. -/
-@[simp] lemma nnnorm_smul_right_apply (c : E →L[𝕜] 𝕜) (f : F) :
+@[simp] lemma nnnorm_smul_right_apply (c : E →L[𝕜] 𝕜) (f : Fₗ) :
   ∥smul_right c f∥₊ = ∥c∥₊ * ∥f∥₊ :=
 nnreal.eq $ c.norm_smul_right_apply f
 
-variables (𝕜 E F)
+variables (𝕜 E Fₗ)
 
 /-- `continuous_linear_map.smul_right` as a continuous trilinear map:
 `smul_rightL (c : E →L[𝕜] 𝕜) (f : F) (x : E) = c x • f`. -/
-def smul_rightL : (E →L[𝕜] 𝕜) →L[𝕜] F →L[𝕜] E →L[𝕜] F :=
+def smul_rightL : (E →L[𝕜] 𝕜) →L[𝕜] Fₗ →L[𝕜] E →L[𝕜] Fₗ :=
 linear_map.mk_continuous₂
   { to_fun := smul_rightₗ,
-    map_add' := λ c₁ c₂, by { ext x, simp [add_smul] },
-    map_smul' := λ m c, by { ext x, simp [smul_smul] } }
-  1 $ λ c x, by simp
-
-variables {𝕜 E F}
-
-@[simp] lemma norm_smul_rightL_apply (c : E →L[𝕜] 𝕜) (f : F) :
-  ∥smul_rightL 𝕜 E F c f∥ = ∥c∥ * ∥f∥ :=
+    map_add' := λ c₁ c₂, by
+    { apply linear_map.ext,
+      intro x,
+      apply continuous_linear_map.ext,
+      intro y,
+      simp only [add_smul, coe_smul_rightₗ, add_apply, smul_right_apply, linear_map.add_apply] },
```

Comment:
Why did you change the proof? As far as I can tell, the old one is still working. If you find it too slow, could you just squeeze the simp?

Reply:
Yes, this was a performance issue -- I initially thought `ext` was the problem, but indeed `simp` was the main one.